astropy/table/table.py

```diff
@@ -150,6 +150,39 @@ def keys(self):
     def values(self):
         return list(OrderedDict.values(self))
 
+    def isinstance(self, *cls):
```

Comment:
Ideally this method will be used internally as we've done. But if at all the user wants to retrieve columns of a specific class, probably to check if there is a particular mixin or not, then he could do so. Do tell me what is required.

Regarding *cls, tuple sounds good. Tell me about this too.